### PR TITLE
hide language/theme

### PR DESCRIPTION
**Pull request by ximex**
hide language/theme menu if not more than 1 selection is available

ready for review/merge

### CODE REVIEW

File: resources/views/layouts/parts/navbar.twig [hunk 1]
```diff
@@ -92,14 +92,16 @@
...
 
-                <li class="nav-item dropdown">
-                    <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-                        {{ m.icon('translate') }}
-                    </a>
-                    <ul class="dropdown-menu dropdown-menu-end">
-                        {{ menuLanguages()|join(" ")|raw }}
-                    </ul>
-                </li>
+                {% if showLanguagesMenu() %}
```
Comment (MyIgel) [inline resources/views/layouts/parts/navbar.twig:95]: As this function is only used once i would just directly use `config('locales') | length > 1`

File: src/Controllers/SettingsController.php [hunk 2]
```diff
@@ -243,10 +243,16 @@ public function settingsMenu(): array
         $menu = [
             url('/settings/profile')  => 'settings.profile',
             url('/settings/password') => 'settings.password',
-            url('/settings/language') => 'settings.language',
-            url('/settings/theme')    => 'settings.theme',
         ];
 
+        if (count(config('locales')) >= 2) {
```
Comment (MyIgel) [inline src/Controllers/SettingsController.php:248]: Imho `> 1` would be easier to read/understand here

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: src/Controllers/SettingsController.php [hunk 3]
```diff
@@ -243,10 +243,16 @@ public function settingsMenu(): array
...
             url('/settings/password') => 'settings.password',
-            url('/settings/language') => 'settings.language',
-            url('/settings/theme')    => 'settings.theme',
         ];
 
+        if (count(config('locales')) > 1) {
+            $menu[url('/settings/language')] = 'settings.language';
+        }
+
+        if (count(config('themes')) >= 2) {
```
Comment (MyIgel) [inline src/Controllers/SettingsController.php:252]: Here too (as its even more confusing that way) ;)

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: src/Controllers/SettingsController.php (hunk 3 above)
Comment (ximex) [reply src/Controllers/SettingsController.php]: fixed

**Pull request merged**